test/composables/useSidebar.itemsByPaths.test.ts

```
@@ -701,9 +701,9 @@ describe('useSidebar itemsByPaths', () => {
     ])
   })
 
-  it('generate items witn sidebarGroupTemplate', () => {
+  it('generate items with sidebarGroupTemplate', () => {
     const result = sidebar.itemsByPaths({
-      sidebarGroupTemplate: (path: string, depth: number) => `${depth} - ${path}`,
+      sidebarGroupTemplate: (path: string, depth: number = 1) => `${depth} - ${path}`,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)